views/incubator/omniledger.tpl

```diff
@@ -0,0 +1,101 @@
+            In the case of Bitcoin, transactions describe the <strong>movement of money</strong>
+            between different parties.
+            Instead of having a central bank deciding which money transfers are valid,
+            Bitcoin spreads this decision over many computers which agree together which transfers are valid.
+            A new kind of transaction that is gaining traction in 2021 are Non Fungible Tokens, or NFTs,
+            which represent the digital ownership of a physical object.
+        </td>
+        <td width="33%" style="padding: 1em;">
+            <h3>Consensus</h3>
+            <img src="/resources/incubator/images/omniledger/distributed-ledger.png" width="33%"
+                 style="float: left; padding: 1em;"/>
+            There are other blockchains than Bitcoin, but they all solve the problem of
+            <strong>consensus</strong> for decentralized systems. Reaching a consensus between
+            different entities is not easy, as some of the entities can try to cheat.
+            So the conception of blockchains always takes into account that some of the
+            participants will try to cheat. If too many are cheating, the consensus will fail,
+            or invalid transactions might get accepted, leading to a failure of the
+            decentralization.
+        </td>
+    </tr>
+</table>
+
+<h2>OmniLedger</h2>
+
+<p>
+    When building blockchains, there are a number of technical challenges to overcome.
+    OmniLeger solves the following challenges:
+</p>
+<ul>
+    <li><strong>Speed</strong> - if many people want to use the system, it needs to be fast. Speed is measured
+        in two ways: transactions per second, and confirmation delay
+    </li>
+    <li><strong>Transactions per second</strong> - define how many money transfers can be processed by the system.
+        VISA can process up to 10'000 transactions in a second, while Bitcoin can do about 7.
+    </li>
+    <li><strong>Confirmation delay</strong> - how long you have to wait to be sure your money transfer arrived.
+        VISA takes about 5 seconds, Bitcoin takes about 1h.
+    </li>
+    <li><strong>Attack resistance</strong> - if you receive money in the system, you want to make sure it
+        stays there, and does not disappear all of a sudden.
+    </li>
+    <li><strong>Secret storage</strong> - today's blockchains are often world-readable, so it is important
+    that you still can store secret data without everybody being able to read it.</li>
+</ul>
+
+<h2>Links to demos</h2>
+
+<p>OmniLedger has already been used for a number of years, and there are a lot of projects that
+interact with this blockchain.
+Together with the <a href="https://dedis.ch">DEDIS</a> lab, C4DT is running a test-network for
+more than a year now, and it is used on a daily basis for different services.</p>
+
+<ul>
+    <li><a href="https://wookie.ch/columbus">Blockchain Explorer</a> -
```

Comment:
```suggestion
    <li><a href="https://wookiee.ch/columbus">Blockchain Explorer</a> -
```

Reply:
Thanks. that should _really_ be somewhere under the `dedis.ch` domain...